zipline-loader/src/commonMain/kotlin/app/cash/zipline/loader/ZiplineCache.kt

```diff
@@ -80,27 +80,23 @@ class ZiplineCache internal constructor(
     driver.close()
   }
 
-  internal fun write(
+  private fun write(
     applicationName: String,
     sha256: ByteString,
     content: ByteString,
     nowEpochMs: Long,
     isManifest: Boolean = false,
     manifestFreshAtMs: Long? = null,
-  ): Files? {
-    try {
```

Comment:
lots of the diffs in this file are moving try/catch blocks outside of implementation-detail functions and towards the public API. They’re more coarse-grained that way and that’s okay.